tests.py

```diff
@@ -747,5 +770,5 @@ def test_delete(self):
 
 
 if __name__ == '__main__' and __package__ is None:
-    __package__ = "toggl"
+    __package__ = 'toggl'
```

Comment:
Gah, I see why people use formatters—the inconsistency here is a little maddening (considering how hard I tried to back out my formatting changes).